sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1708,6 +1708,13 @@ object SQLConf {
       "and java.sql.Date are used for the same purpose.")
     .booleanConf
     .createWithDefault(false)
+
+  val SQL_EVENT_TRUNCATE_LENGTH = buildConf("spark.sql.event.truncate.length")
```

Comment:
Hi, @ajithme .
- Can we move this to `StaticSQLConf` like `spark.sql.ui.retainedExecutions` configuration?
- Naming is always difficult. For config name, I'd suggest `spark.eventLog.sql.maxLength`. Let's get some comment after updating to that.

Reply:
@dongjoon-hyun 
1. Yes. I have moved it to StaticSQLConf 
2. i dint want to mention ``eventLog`` in the configuration because this property will truncate SQL in the ``SparkListenerSQLExecutionStart`` creation itself. ``EventLog`` is just one of the Listener to the event and infact all listeners of this event will get the truncated SQL.  right.?